test/box/graceful_shutdown.test.lua

```diff
@@ -0,0 +1,91 @@
+net_box = require('net.box')
```

Comment:
Test is rather long (about 7 - 10 seconds). @Korablev77 it's ok? I think may be reduce sleep time? Also i think it will be good to add more comments to make test more clear. May be write an explanation of what you are testing at a particular moment.